[Jul 22 20:56:05] INFO[16577]: asterisk.test_case:173 __init__: Executing tests/channels/pjsip/subscriptions/rls/lists/off_nominal/presence/no_list_resources_exist
[Jul 22 20:56:05] INFO[16577]: asterisk.test_case:278 create_asterisk: Creating Asterisk instance 1
[Jul 22 20:56:06] INFO[16577]: asterisk.test_case:444 _start_asterisk: Starting Asterisk instance 1
[Jul 22 20:56:08] INFO[16577]: asterisk.test_case:329 create_ami_factory: Creating AMIFactory 1 to 127.0.0.1
[Jul 22 20:56:08] INFO[16577]: twisted:154 publishToNewObserver: Starting factory <starpy.manager.AMIFactory instance at 0x7fa48658fd88>
[Jul 22 20:56:08] INFO[16577]: AMI:184 connectionMade: Connection Made
[Jul 22 20:56:08] INFO[16577]: AMI:203 onComplete: Login Complete: {u'message': u'Authentication accepted', u'response': u'Success', u'actionid': u'd7a6f074b43c-140344605340160-1'}
[Jul 22 20:56:08] INFO[16577]: asterisk.test_case:582 _ami_connect: AMI Connect instance 1
[Jul 22 20:56:08] INFO[16577]: asterisk.sipp:702 run: Executing SIPp scenario: list_subscribe.xml
[Jul 22 20:56:08] WARNING[16577]: asterisk.sipp:524 processEnded: Resolving remote host '127.0.0.1'... Done.

[Jul 22 20:56:08] WARNING[16577]: asterisk.sipp:524 processEnded: 2018-07-22	20:56:08.257428	1532307368.257428: Aborting call on unexpected message for Call-Id '1-16662@127.0.0.1': while expecting '100' (index 1), received 'SIP/2.0 404 Not Found
Via: SIP/2.0/UDP 127.0.0.1:5061;rport=5061;received=127.0.0.1;branch=z9hG4bK-16662-1-0
Call-ID: 1-16662@127.0.0.1
From: "sipp" <sip:sipp@127.0.0.1>;tag=16662SIPpTag001
To: "sut" <sip:pres_list@127.0.0.1>;tag=z9hG4bK-16662-1-0
CSeq: 1 SUBSCRIBE
Server: Asterisk PBX GIT-16-9767e98486
Content-Length:  0

'.

[Jul 22 20:56:08] WARNING[16577]: asterisk.sipp:628 __scenario_callback: SIPp Scenario list_subscribe.xml Failed [1]
[Jul 22 20:56:08] INFO[16577]: asterisk.sipp:198 _handle_scenario_finished: Scenario list_subscribe.xml passed
[Jul 22 20:56:08] INFO[16577]: asterisk.sipp:217 _finish_test: All SIPp Scenarios executed; stopping reactor
[Jul 22 20:56:08] INFO[16577]: asterisk.test_case:487 __stop_instances: Stopping Asterisk instance 1
[Jul 22 20:56:08] WARNING[16577]: asterisk.sipp:637 __evaluate_scenario_results: SIPp Scenario list_subscribe.xml Failed
[Jul 22 20:56:08] INFO[16577]: asterisk.sipp:532 processEnded: SIPp scenario list_subscribe.xml ended with code 1
[Jul 22 20:56:08] INFO[16577]: AMI:1136 clientConnectionLost: connection lost, reconnecting...
[Jul 22 20:56:08] INFO[16577]: twisted:154 publishToNewObserver: <twisted.internet.tcp.Connector instance at 0x7fa48658fe18> will retry in 2 seconds
[Jul 22 20:56:08] INFO[16577]: AMI:1139 clientConnectionLost: None
[Jul 22 20:56:08] INFO[16577]: twisted:154 publishToNewObserver: Stopping factory <starpy.manager.AMIFactory instance at 0x7fa48658fd88>
[Jul 22 20:56:09] INFO[16577]: asterisk.test_case:508 __stop_reactor: Stopping Reactor
[Jul 22 20:56:09] INFO[16577]: twisted:154 publishToNewObserver: Main loop terminated.
[Jul 22 20:56:09] INFO[16577]: test_runner:308 main: Test run for tests/channels/pjsip/subscriptions/rls/lists/off_nominal/presence/no_list_resources_exist completed with result False
